### PR TITLE
Organizing

### PR DESCRIPTION
**Pull request by mastermind-mayhem**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -74,12 +69,7 @@ public class RobotContainer {
   // // Wrist
   private final LowerCmd lowerCmd = new LowerCmd(wristSubsystem);
   private final RaiseCmd raiseCmd = new RaiseCmd(wristSubsystem);
-  // private final RotLeftCmd rotLeftCmd = new RotLeftCmd(wristSubsystem);
-  // private final RotRightCmd rotRightCmd = new RotRightCmd(wristSubsystem);
   private final StopRaiseCmd stopRaiseCmd = new StopRaiseCmd(wristSubsystem);
-  // private final StopRotCmd stopRotCmd = new StopRotCmd(wristSubsystem);
-  // private final RotLeft90Cmd rotLeft90Cmd = new RotLeft90Cmd(wristSubsystem);
-  // private final RotRight90Cmd rotRight90Cmd = new RotRight90Cmd(wristSubsystem);
```
[the remark this answers is not in the document]
Comment (mastermind-mayhem) [reply src/main/java/frc/robot/RobotContainer.java:82]: I did....

[the remark this answers is not in the document]
Comment (mathyourlife) [reply src/main/java/frc/robot/RobotContainer.java:82]: my bad :D

**Comment by kv-gh** [comment]
Quick question as a point of curiosity: I've noticed a couple diffs where we are adding imports to files where there are no corresponding code changes. Does that mean the imports are unused? Or are we adding them because they were missing before?

**Comment by mastermind-mayhem** [comment]
> Quick question as a point of curiosity: I've noticed a couple diffs ([example](https://github.com/OysterRiverOverdrive/Charged-Up-2023-Atlas_Chainsaw/pull/79/files#diff-28265b964ae65aa31934668adf5caace028040f58954dcd46f7218b81a83d342R8)) where we are adding imports to files where there are no corresponding code changes. Does that mean the imports are unused? Or are we adding them because they were missing before?

**Pull request closed without merging**